### DISCUSSION TITLE
Specify augmentation for different images

### DISCUSSION THREAD

**Discussion by rousseau** [TorchIO-project/torchio#536] [Q&A] [answered]
First, thanks for this great tool !

I have a question regarding the data augmentation strategy. If for each subject, there is one image (T1). Is it possible to apply two different augmentations ? 

For instance :
In a case of supervised denoising method, 
training_transform_T1a = tio.Compose([spatial, noise])
training_transform_T1b = tio.Compose([spatial])
(the spatial deformations are the same for both images)

Or should we use the "include" option of the transform object and maybe specify 2 images for each subject ?

**Comment by fepegar** [comment, marked as answer]
Hi, @rousseau. I believe you can use `include`/`exclude` for this.

```python
import torchio as tio
transform = tio.Compose([
    tio.RandomAffine(),
    tio.RandomBlur(std=(2, 3), include='t1'),
])
subject = tio.datasets.Pediatric(years=(4.5, 8.5))
subject.remove_image('pd')
subject.remove_image('mask')
subject.plot(show=False)
transformed = transform(subject)
transformed.plot()
```

![Figure_1](https://user-images.githubusercontent.com/12688084/118294467-a76cf680-b4d2-11eb-9e28-e765d5e54fa0.png)
![Figure_2](https://user-images.githubusercontent.com/12688084/118294487-aa67e700-b4d2-11eb-8589-bb0c27f330ff.png)

**Reply by rousseau** [reply]
Thank a lot.

**Reply by fepegar** [reply]
No worries!
